bigtop_toolchain/manifests/packages.pp

```diff
@@ -348,6 +348,37 @@
       }
     }
   }
+  if $operatingsystem == 'Rocky' {
+    package { 'epel-release':
+      ensure => installed,
+      notify => Package[$pkgs]
+    }
+    if versioncmp($operatingsystemmajrelease, '8') == 0 {
+      # On Rocky 8, EPEL requires that the PowerTools repository is enabled.
+      # See https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F
+      yumrepo { 'powertools':
+        ensure  => 'present',
+        enabled => '1'
+      }
+      yumrepo { 'devel':
+        ensure  => 'present',
+        enabled => '1'
+      }
+      Yumrepo<||> -> Package<||>
+    }
+    if versioncmp($operatingsystemmajrelease, '9') == 0 {
+      # On Rocky 9, EPEL requires that the crb repository is enabled.
+      yumrepo { 'crb':
+        ensure  => 'present',
+        enabled => '1'
+      }
+      yumrepo { 'devel':
+        ensure  => 'present',
+        enabled => '1'
+      }
+      Yumrepo<||> -> Package<||>
+    }
+  }
```

Comment:
@masatana Could you add equivalent of this to the manifest of bigtop-deploy too? Some packages (e.g. python3-devel on rockylinux-8) seem to need this. We are using bigtop-deploy over bigtop/puppet images on CI.

Reply:
@iwasakims Thanks for the comment. Could you check the updated patch?